examples/cloudflare-workers/README.md

```diff
@@ -28,13 +28,7 @@ npm install @upstash/rag-chat
 Ensure your wrangler.toml file includes the following configuration to enable Node.js compatibility:
 
 ```toml
-node_compat = true
-```
-
-In older CF worker versions, you may need to set the following compatibility flags:
-
-```toml
-compatibility_flags = [ "streams_enable_constructors", "transformstream_enable_standard_constructor" ]
```

Comment:
why do we delete these? I needed them for streaming, did you try streaming with only the nodejs_compat_v2 flag?

Reply:
Yes it works, why did you need them, shouldn't they be default?
https://developers.cloudflare.com/workers/configuration/compatibility-dates/#streams-constructors
https://developers.cloudflare.com/workers/configuration/compatibility-dates/#compliant-transformstream-constructor

Reply:
`In older CF worker versions` :D

let's delete them then, if someone has any issues, we can let them know about the flags.

Reply:
Ohh makes sense, I will add it back